### ISSUE TITLE
Sometimes read on scroll is broken

### ISSUE THREAD

**Issue by Jenselme** [Jenselme/legadilo#195]
No information on why. It seems erratic: when it happens, I didn’t do anything specific. Maybe I should try to improve logging of the feature to at least know what is going wrong with it.

**Comment by Jenselme** [comment]
Console logs (no network tab info sadly):
```
Read on scroll setup! [list_of_articles.a7e8277c66a1.js:74:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Done for https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/, resolving promise. [list_of_articles.a7e8277c66a1.js:102:19](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Previous promise resolved, starting https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ [list_of_articles.a7e8277c66a1.js:94:13](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Form submission via untrusted submit event is deprecated and will be removed at a future date. [htmx.min.fda0965a21d4.js:1:26116](https://www.legadilo.eu/static/htmx.min.fda0965a21d4.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
Adding https://www.legadilo.eu/reading/articles/X/MARK_AS_READ/ to the promise chain. [list_of_articles.a7e8277c66a1.js:88:15](https://www.legadilo.eu/static/js/list_of_articles.a7e8277c66a1.js)
```

Besides the `Form submission via untrusted submit event is deprecated and will be removed at a future date.` error message, nothing looks spurious.

**Comment by Jenselme** [comment]
It happened again. Same error message.

**Comment by Jenselme** [comment]
It seems to be linked to firefox. Hard to reproduce on test data. Let’s gather more examples and see.

**Comment by Jenselme** [comment]
This seems to have been solved by #250 Closing.
